opentelemetry-exporter-gcp-logging/tests/__snapshots__/test_cloud_logging/test_agent_engine_monitored_resources[client].json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "entries": [
+      {
+        "logName": "projects/fakeproject/logs/test",
+        "resource": {
+          "labels": {
```

Comment:
Verified the logging API accepts this 

<img width="320" height="99" alt="Image" src="https://github.com/user-attachments/assets/e15c2ed4-442c-4f83-87ca-d8b3d6ece39a" />